opaque_keys/edx/keys.py

```diff
@@ -61,6 +61,13 @@ def make_asset_key(self, asset_type, path):  # pragma: no cover
         """
         raise NotImplementedError()
 
+    @abstractmethod
+    def make_course_key_v2(self):  # pragma: no cover
```

Comment:
Is this really necessary given that you already have `CourseKeyV2.from_course_run_key()`?